src/components/NcAvatar/NcAvatar.vue

```diff
@@ -710,28 +707,38 @@ export default {
 
 	&--with-menu {
 		cursor: pointer;
-		:deep(.v-popper) {
+		.action-item {
 			position: absolute;
 			top: 0;
 			left: 0;
 		}
-		.icon-more {
+		:deep(.action-item__menutoggle) {
 			cursor: pointer;
 			opacity: 0;
 		}
 		&:focus,
-		&:hover {
-			.icon-more {
+		&:hover,
+		&#{&}-loading {
```

Comment:
What's that magic woodoo selector doing? 🪄

Reply:
`&` - parent selector, equals to `.avatardiv--with-menu`
`#{expression}` - interpolation, `#{$}` equals to value of `&`

So
```scss
.avatardiv {
    &--with-menu {
        &#{&}-loading {
            /* ... */
        }
    }
}
```

compiles to

```scss
.avatardiv--with-menu.avatardiv--with-menu-loading {
  /* ... */
}
```